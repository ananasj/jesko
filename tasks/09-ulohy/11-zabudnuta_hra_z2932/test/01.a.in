5
17 24 11 13 12
19 7 15 8 9
22 2 18 3 21
6 10 14 20 0
23 1 5 4 16
5
23
17
16
5
24
