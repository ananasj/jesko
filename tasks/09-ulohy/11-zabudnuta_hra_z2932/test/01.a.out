17 24 11 13 12
19 7 15 8 9
22 2 18 3 21
6 10 14 20 16
23 1 5 4 0
